### DISCUSSION TITLE
[Kavita] Translation Relationship

### DISCUSSION THREAD

**Discussion by scare376** [Kareadita/Kavita#2959] [Ideas]
### Idea Description

It would be nice to have a translation relationship. I have a few series where I have it in 2 languages. How I imagine it working is you would set it from the original series and it would set the parent relationship tag on the translated series.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

2 years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Wouldn't Edition satisfy the Translation relationship? From my notes, Edition could be colored version, different translation, scan/digital releases.

**Reply by scare376** [reply]
Just saw this. Yes it could work. The issue I have with it is I need to add the relationship to both series individually and preferably there should be some sort of indicator that one is the source language as well.

**Reply by majora2007** [reply]
But wouldn't Parent indicate that it's the source language?

**Reply by scare376** [reply]
Yes it would. That's why I'm saying using edition is an issue. Because it doesn't add that parent relationship.

**Reply by Roshri** [reply]
I agree with OP, I have currently set stuff using Edition because it's the closest thing we have, and that would be fine for two different translated versions, i.e. a Spanish AND English translation of the same manga, those are two different editions, as I see it.
But if I add the Japanese version of the same series, that's the source for both translations and should be the parent for all the different languages.

**Comment by adamradocz** [comment]
I'd love this feature too. For me, edition and translation are different things. A book could have different editions like 1st and 2nd, and also translations of the editions.

**Comment by newyorker123** [comment]
It'll be great to have a translation relationship that shows the actual language on the card, instead of the word "edition".
